nautobot_secrets_providers/providers/hashicorp.py

```diff
@@ -24,19 +24,26 @@
 AUTH_METHOD_CHOICES = ["approle", "aws", "kubernetes", "token"]
 
 
-# Default mount point for the HVAC client
-try:
-    plugins_config = settings.PLUGINS_CONFIG["nautobot_secrets_providers"]
```

Comment:
I feel like this was done so that a default would be provided. I went back and forth on this, those defaults were good when their was only one configuration, but now that you have many configurations, you kind of need a default per configuration.